app/src/main/java/com/ncnf/home/ui/HomeFragment.java

```diff
@@ -63,8 +63,7 @@ private void GoTo(Class<?> activity){
     }
 
     public void goToFriends(View view){
-        Intent intent = new Intent(getContext(), FriendsActivity.class);
-        startActivity(intent);
+        GoTo(FriendsActivity.class);
```

Comment:
Method that starts with a capital letter :O

Reply:
Agreed, this changed.